<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>502</width>
     <height>256</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="1">
       <widget class="QPushButton" name="buttonLn">
        <property name="text">
         <string>ln</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="buttonInvers">
        <property name="text">
         <string>1/x</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="buttonPow">
        <property name="text">
         <string>x^y</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="buttonSqr">
        <property name="text">
         <string>xÂ²</string>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QPushButton" name="buttonDot">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QPushButton" name="button0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="buttonSign">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="button7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QPushButton" name="button8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QPushButton" name="button9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QPushButton" name="button6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="button5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="button4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QPushButton" name="button3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="button2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="button1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="mcButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>MC</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="mrButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>MR</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="mButton">
        <property name="text">
         <string>M</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QRadioButton" name="radians">
        <property name="text">
         <string>Radians</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QRadioButton" name="grads">
        <property name="text">
         <string>Grads</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QRadioButton" name="degrees">
        <property name="text">
         <string>Degrees</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="buttonInv">
        <property name="text">
         <string>INV</string>
        </property>
       </widget>
      </item>
      <item row="7" column="6">
       <widget class="QPushButton" name="buttonD">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="QPushButton" name="buttonT">
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QPushButton" name="buttonM">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QPushButton" name="buttonA">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QPushButton" name="buttonDel">
        <property name="text">
         <string>Del</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QPushButton" name="buttonClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4" colspan="2">
       <widget class="QPushButton" name="buttonEq">
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="buttonCos">
        <property name="text">
         <string>cos</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="buttonSin">
        <property name="text">
         <string>sin</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="buttonTan">
        <property name="text">
         <string>tan</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="buttonLog">
        <property name="text">
         <string>log</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>10</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
